site/content/faq/800-how-do-i-test-svelte-apps.md

```diff
@@ -2,6 +2,8 @@
 question: How do I test Svelte apps?
 ---
 
+We recommend trying to seperate your view logic from your business logic. Data transformation or cross component state management is best kept outside of Svelte components. You can test those parts like you would test any JavaScript functionality that way. When it comes to testing the components, it is best to test the business logic of the component and remember that the Svelte library has its own tests and you do not need to test implementation details provided by Svelte.
```

Comment:
In the first sentence we recommend to split out the business logic. In a later sentence we use that same word in the context of component tests. I think we should choose a different wording there to not confuse people.

Reply:
updated